contracts/script/deploy/sepolia/Deploy.s.sol

```diff
@@ -0,0 +1,255 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity =0.8.12;
+
+import {EmptyContract} from "lib/eigenlayer-middleware/lib/eigenlayer-contracts/src/test/mocks/EmptyContract.sol";
+import {ProxyAdmin, TransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+import {IDelegationManager} from
+    "lib/eigenlayer-middleware/lib/eigenlayer-contracts/src/contracts/interfaces/IDelegationManager.sol";
+import {ISocketRegistry, SocketRegistry} from "lib/eigenlayer-middleware/src/SocketRegistry.sol";
+import {IIndexRegistry} from "lib/eigenlayer-middleware/src/interfaces/IIndexRegistry.sol";
+import {IndexRegistry} from "lib/eigenlayer-middleware/src/IndexRegistry.sol";
+import {IStakeRegistry, StakeRegistry} from "lib/eigenlayer-middleware/src/StakeRegistry.sol";
+import {IBLSApkRegistry} from "lib/eigenlayer-middleware/src/interfaces/IBLSApkRegistry.sol";
+import {BLSApkRegistry} from "lib/eigenlayer-middleware/src/BLSApkRegistry.sol";
+import {RegistryCoordinator, IRegistryCoordinator} from "lib/eigenlayer-middleware/src/RegistryCoordinator.sol";
+import {IEigenDAThresholdRegistry, EigenDAThresholdRegistry} from "src/core/EigenDAThresholdRegistry.sol";
+import {IEigenDARelayRegistry, EigenDARelayRegistry} from "src/core/EigenDARelayRegistry.sol";
+import {PaymentVault} from "src/payments/PaymentVault.sol";
+import {IPaymentVault} from "src/interfaces/IPaymentVault.sol";
+import {IEigenDADisperserRegistry, EigenDADisperserRegistry} from "src/core/EigenDADisperserRegistry.sol";
+import {EigenDAServiceManager, IServiceManager} from "src/core/EigenDAServiceManager.sol";
+import {IAVSDirectory} from
+    "lib/eigenlayer-middleware/lib/eigenlayer-contracts/src/contracts/interfaces/IAVSDirectory.sol";
+import {IRewardsCoordinator} from
+    "lib/eigenlayer-middleware/lib/eigenlayer-contracts/src/contracts/interfaces/IRewardsCoordinator.sol";
+import {IServiceManager} from "src/core/EigenDAServiceManager.sol";
+import {VersionedBlobParams} from "src/interfaces/IEigenDAStructs.sol";
+
+import {MockStakeRegistry} from "./mocks/MockStakeRegistry.sol";
+import {MockRegistryCoordinator} from "./mocks/MockRegistryCoordinator.sol";
+
+import {
+    DeploymentInitializer,
+    ImmutableInitParams,
+    DeployedAddresses,
+    ImmutableRegistryCoordinatorParams,
+    ImmutablePaymentVaultParams,
+    ImmutableServiceManagerParams,
+    CalldataInitParams,
+    CalldataRegistryCoordinatorParams,
+    CalldataThresholdRegistryParams,
+    CalldataServiceManagerParams
+} from "./DeploymentInitializer.sol";
+
+import "forge-std/Script.sol";
+import "forge-std/StdJson.sol";
+
+contract DeploySepolia is Script {
+    string deployConfigPath = string(bytes("./script/deploy/sepolia/config/placeholder.config.json"));
```

Comment:
Do you not like forge's best practice of having a single deploy file that works with config files for different networks? See #7 in https://book.getfoundry.sh/guides/best-practices#scripts

Do you think its safer to have a Deploy Script per network with hardcoded config path like this?

Reply:
As @ethenotethan mentions, it should be possible to make this all chain agnostic. I'll work on that.